model/reward/instructor/rank_datasets.py

```diff
@@ -261,11 +261,16 @@ def preprocess_dialogue(self, text):
             return "Human:".join(last_two_convo)
         return "Human: " + "Human:".join(last_two_convo)
 
-    def __init__(self, split="train", sep_token="<sep>") -> None:
+    def __init__(self, split="train", sep_token="<sep>", mode=None) -> None:
         super().__init__()
+        self.mode = mode
         assert split in ("train", "test")
         if sep_token is None:
             sep_token = " . "
+
+        if mode == "rm":
```

Comment:
This are changes in the reward/instructor folder. They are not used for the trainer_rm code. I suggest to remove these changes.